themes/default/layouts/partials/registry/package.html

```diff
@@ -66,13 +72,38 @@ <h3 class="text-gray-900 no-anchor">
                     {{ end }}
 
                     <!-- Fixed-height spacer accounts for the minimum space required by the tags. -->
-                    <div class="h-14"></div>
+                    <div class="spacer h-14"></div>
 
                     <!-- Tags are anchored to the bottom of the card. -->
                     <ul class="package-tags mt-2 absolute bottom-8">
                         <li class="package-tag-type" data-filter-group="type" data-filter-value="{{ $packageType }}">{{ $packageTypeName }}</li>
                         <li class="package-tag-category" data-filter-group="category" data-filter-value="{{ $packageCategory }}">{{ $packageCategoryName }}</li>
                     </ul>
+
+                    <div class="maintainer flex items-center flex-grow justify-end mt-2 absolute bottom-8 h-14">
+                        <div class="text-right">
+                            <pulumi-tooltip>
+                                <div class="flex items-center pt-1">
+                                <span class="pr-2">{{ $packageLabel }}</span>
+                    
+                                {{ if (eq (.package.publisher) "Pulumi") }}
+                                    <img src="/icons/check-verified.svg">
+                                    </div>
```

Comment:
Should this lone closing `div` be here like this? If so, maybe there's a better way to structure the markup such that this asymmetry wouldn't be necessary?